bedrock/foundation/templates/foundation/annualreport/2019/index.html

```diff
@@ -542,8 +542,7 @@ <h2 class="mzp-c-card-feature-title"><span class="highlight green">Mozilla urged
             <img src="{{ static('img/foundation/annualreport/2019/modal-dots-2.svg') }}" alt="">
             <h2 class="mzp-c-card-feature-title"><span class="highlight green">The internet needs our love</span></h2>
             <p class="mzp-c-card-feature-desc">As part of returning Firefox to growth, we have to speak to people’s concerns about online life with our marketing and communications. 2020 revealed more than ever just how fed up internet users are with how their data is used by big platforms, and what little they get in exchange for giving up privacy to them.</p>
-            <p class="mzp-c-card-feature-desc">With this in mind we launched a <a href="{{ url('firefox.campaign.unfck.index') }}">campaign</a> with some easy choices for people to make to unfck their own corner of the internet. We believe those taking part will help drive meaningful change and create a healthier online life for all.</p>
-            <a target="_blank" rel="external noopener" class="mzp-c-button mzp-t-secondary" href="{{ url('firefox.campaign.unfck.index') }}">Learn more</a>
+            <p class="mzp-c-card-feature-desc">With this in mind we launched a campaign with some easy choices for people to make to unfck their own corner of the internet. We believe those taking part will help drive meaningful change and create a healthier online life for all.</p>
```

Comment:
Should we mention here that it's been decommissioned?

Reply:
The campaign was more than just the landing page here, there were lots of different pieces of creative work that went into it. So the copy still applies well I think even without the link.